### PR TITLE
docs: update badges style

### PR DESCRIPTION
**Pull request by Atinux**
And add a link to Volta board (soon won't be the need to login to view it)

Wondering if GitHub CI badge is necessary now?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,14 +59,15 @@ yarn add @nuxtjs/sanity # or npm install @nuxtjs/sanity
...
-
-[npm-version-src]: https://img.shields.io/npm/v/@nuxtjs/sanity/latest.svg
+[npm-version-src]: https://img.shields.io/npm/v/@nuxtjs/sanity/latest.svg?style=flat&colorA=18181B&colorB=28CF8D
 [npm-version-href]: https://npmjs.com/package/@nuxtjs/sanity
-[npm-downloads-src]: https://img.shields.io/npm/dm/@nuxtjs/sanity.svg
+[npm-downloads-src]: https://img.shields.io/npm/dm/@nuxtjs/sanity.svg?style=flat&colorA=18181B&colorB=28CF8D
 [npm-downloads-href]: https://npmjs.com/package/@nuxtjs/sanity
-[github-actions-ci-src]: https://github.com/nuxt-modules/sanity/workflows/ci/badge.svg
+[github-actions-ci-src]: https://github.com/nuxt-modules/sanity/workflows/ci/badge.svg?style=flat&colorA=18181B&colorB=28CF8D
 [github-actions-ci-href]: https://github.com/nuxt-modules/sanity/actions?query=workflow%3Aci
```
Comment (danielroe) [inline README.md:67]:
```suggestion
```

Comment (Atinux) [reply README.md]: Agree

File: README.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 [![Github Actions CI][github-actions-ci-src]][github-actions-ci-href]
```
Comment (danielroe) [inline README.md:7]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -58,14 +58,14 @@ yarn add @nuxtjs/sanity # or npm install @nuxtjs/sanity
...
-[npm-version-src]: https://img.shields.io/npm/v/@nuxtjs/sanity/latest.svg
+[npm-version-src]: https://img.shields.io/npm/v/@nuxtjs/sanity/latest.svg?style=flat&colorA=18181B&colorB=28CF8D
 [npm-version-href]: https://npmjs.com/package/@nuxtjs/sanity
-[npm-downloads-src]: https://img.shields.io/npm/dm/@nuxtjs/sanity.svg
+[npm-downloads-src]: https://img.shields.io/npm/dm/@nuxtjs/sanity.svg?style=flat&colorA=18181B&colorB=28CF8D
 [npm-downloads-href]: https://npmjs.com/package/@nuxtjs/sanity
-[github-actions-ci-src]: https://github.com/nuxt-modules/sanity/workflows/ci/badge.svg
-[github-actions-ci-href]: https://github.com/nuxt-modules/sanity/actions?query=workflow%3Aci
-[codecov-src]: https://img.shields.io/codecov/c/github/nuxt-modules/sanity.svg
+[github-actions-ci-src]: https://github.com/nuxt-modules/sanity/workflows/ci/badge.svg?style=flat&colorA=18181B&colorB=28CF8D
```
Comment (danielroe) [inline README.md:65]:
```suggestion
```

**Review by danielroe — APPROVED** [review]
Thanks! ❤️

**Pull request merged**